pkg/spec/populators_test.go

```diff
@@ -17,11 +17,14 @@ limitations under the License.
 package spec
 
 import (
+	"fmt"
 	"reflect"
 	"sort"
 	"testing"
 
 	api_v1 "k8s.io/client-go/pkg/api/v1"
+
```

Comment:
nit: `s/\n//`

can you arrange the imports similar to the comment here https://github.com/kedgeproject/kedge/pull/149#issuecomment-315051859

Reply:
This is similar to what's done here - https://github.com/kedgeproject/kedge/blob/master/pkg/spec/resources.go#L19-L38 - what's wrong?